docs/en/siem/detection-engine-intro.asciidoc

```diff
@@ -0,0 +1,97 @@
+[[detection-engine-overview]]
+[role="xpack"]
+
+== Detections (Beta)
+
+beta[]
+
+The SIEM Detections feature automatically searches for threats and creates 
+signals when they are detected. Signal detection rules define the conditions 
+for creating signals. The {siem-app} comes with
+<<prebuilt-rules, prebuilt rules>> that search for suspicious 
+activity on your network and hosts. Additionally, you can
+<<rules-ui-create, create your own rules>>.
+
+[IMPORTANT]
+==============
+See <<detections-permissions, {kib} and index privileges>> required for viewing 
+signals and creating rules.
+==============
+
+[float]
+[[det-engine-terminology]]
+=== Terminology
+
+Signals::
+Always refer to a {siem-soln} produced detections. Signals are never 
+received from third-party systems. When a rule's conditions are met, the
+{siem-app} writes one or more signals to an Elasticsearch `signals` index.
+
+[NOTE]
+==============
+Signal indices are created for each {kib} space. The naming convention is:
+`.siem-signals-<space name>`. For the default space, the signals index is named 
+`.siem-signals-default`.
+==============
+
+Alerts and events::
+Always refer to data the {siem-app} receives from data shippers and external 
+tools, such as Suricata and Elastic Endpoint Security.
+
+[float]
+=== Signals and external alerts
+
+The Detections page displays all signals and alerts. To view signals created 
+by a rule, click *Manage signal detection rules* and then the relevant rule in 
+the All rules table.
```

Comment:
Might be too much information for this block, but just want to note you can also use the KQL filter to narrow which signals are displayed on the main `Detections page` ala:

```
signal.rule.name: "SSH (Secure Shell) from the Internet" 
```

Only on the main `Detections page`, `Rule Details`, and Timelines that have `All events` or `Signal events` selected do we add the `.siem-signals` index. So it's only on those two pages and applicable Timelines that the user will get autosuggestions when typing `signal.`.